compiler/ras/LimitFile.cpp

```diff
@@ -441,7 +441,8 @@ TR_Debug::inlinefileOption(char *option, void *base, TR::OptionTable *entry, TR:
       }
    else
       {
-      TR_VerboseLog::write("<JIT: warning: unable to read inline file --> '%s' will be ignored>\n", inlineFileName);
+      TR_VerboseLog::write("<JIT: fatal: unable to read inline file --> '%s'>\n", inlineFileName);
+      return option; // This is how to make the JIT fail to initialize. We want to fail if we can't read the file because it is too easy to miss that the file wasn't picked up
```

Comment:
This comment seems like something that belongs in a Doxygen comment as it is non-obvious and as there are multiple exit points of this function. Would it be possible to Doxygen the these two functions and move this comment up to the documentation section?

Reply:
Absolutely. I'll get on that today.